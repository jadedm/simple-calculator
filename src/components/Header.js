import React from 'react';

const Header = () => {
	return(
		<h1 style={{ textAlign: 'center'}}>Simple Calculator</h1>
	);
}

export default Header;