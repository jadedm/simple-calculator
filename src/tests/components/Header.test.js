import React from 'react';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

// import ReactShallowRenderer from 'react-test-renderer/shallow';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Header from './../../components/Header';

configure({ adapter: new Adapter() });
test('should render Header component correctly', () => {
	// const renderer = new ReactShallowRenderer();
	// renderer.render(<Header />);
	// expect(renderer.getRenderOutput()).toMatchSnapshot();

	const wrapper = shallow(<Header />);
	expect(toJSON(wrapper)).toMatchSnapshot();
});