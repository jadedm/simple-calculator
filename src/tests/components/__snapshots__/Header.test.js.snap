// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header component correctly 1`] = `
<h1
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  Simple Calculator
</h1>
`;
